### ISSUE TITLE
Compiling tex sources fails

### ISSUE THREAD

**Issue by jan-warchol** [MaciekPytel/python-memory-talk#2]
Hello,

I've installed latex-beamer and tried creating a pdf with `latex slides.tex` but I got

```
(/usr/share/texmf/tex/latex/beamer/base/translator/dicts/translator-basic-dicti
onary/translator-basic-dictionary-English.dict)
(/usr/share/texmf/tex/latex/beamer/base/translator/dicts/translator-bibliograph
y-dictionary/translator-bibliography-dictionary-English.dict)
(/usr/share/texmf/tex/latex/beamer/base/translator/dicts/translator-environment
-dictionary/translator-environment-dictionary-English.dict)
(/usr/share/texmf/tex/latex/beamer/base/translator/dicts/translator-months-dict
ionary/translator-months-dictionary-English.dict)
(/usr/share/texmf/tex/latex/beamer/base/translator/dicts/translator-numbers-dic
tionary/translator-numbers-dictionary-English.dict)
(/usr/share/texmf/tex/latex/beamer/base/translator/dicts/translator-theorem-dic
tionary/translator-theorem-dictionary-English.dict)
No file slides.nav.
[1] [2] (/usr/share/texmf-texlive/tex/latex/psnfss/t1pcr.fd)
(/usr/share/texmf-texlive/tex/latex/amsfonts/umsa.fd)
(/usr/share/texmf-texlive/tex/latex/amsfonts/umsb.fd) [3]
No file slides.toc.
[4] [5] [6] [7] [8] [9] [10] [11]
Overfull \hbox (103.96982pt too wide) in paragraph at lines 126--126
 [][][][][] 
[12] [13] [14] (./slides.vrb) [15] (./slides.vrb) [16] [17]

! LaTeX Error: Cannot determine size of graphic in gc_tasks.png (no BoundingBox
).

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              

l.188     \end{frame}
```

What should i do to complile the sources?

**Comment by MaciekPytel** [comment]
According to this http://tex.stackexchange.com/questions/17734/cannot-determine-size-of-graphic `latex` doesn't seem able to read the size of image. I'm always using `pdflatex` and it works fine for me.

I got the exact same error when I tried `latex` instead of `pdflatex`, so the stackexchange answer is probably right. This is really annoying - I think writing image sizes in documents is ugly and I don't want to do it. I think I'll just add a makefile to remove any confusion as to how I build the slides.
